includes/class-wc-stripe-order-helper.php

```diff
@@ -829,4 +1017,20 @@ protected function is_order_refund_locked( WC_Order $order ): bool {
 
 		return false;
 	}
+
+	/**
+	 * Updates order meta data.
```

Comment:
Not blocking: It may be worth being explicit as to why this is a helper function.
```suggestion
	 * Helper function to update order meta data. The goal of the function is to reduce boilerplate in the helper due to `null` checks everywhere.
```